packages/cactus-plugin-ledger-connector-polkadot/README.md

```diff
@@ -0,0 +1,67 @@
+# `@hyperledger/cactus-plugin-ledger-connector-polkadot`
```

Comment:
Please refer to other plugins readme for reference (Example: https://github.com/hyperledger/cacti/blob/main/packages/cactus-plugin-ledger-connector-fabric/README.md)

Reply:
Also, you can add the referenced UML diagrams here itself so that I can review it all together

Reply:
done